twitterFederation/services/comments/src/resolvers/mutation.ts

```diff
@@ -0,0 +1,17 @@
+import Comment from '../models/comment';
+
+export default {
+  addComment: async (_: any, data: any) => {
+    const comment = await Comment.create(data.comment);
+    return comment;
+  },
```

Comment:
You could do an [implicit return](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions#function_body) like this:
```suggestion
  addComment: (_: any, data: any) => Comment.create(data.comment),
```